site/gatsby-site/src/components/landing/sponsors.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "name": "Organization Founding Sponsor",
+        "items": [
+            {
+                "name": "Waking Up Foundation",
+                "modalName": "WU",
+                "logo": "wu-foundation-blue-logo.svg",
+                "text": "On the recommendation of Longview Philanthropy, the [[Waking Up Foundation]] contributed a founding grant of $550k USD directed to the AI Incident Database. $33k of the total is committed to our fiscal sponsor for running the books and tax compliance for the organization. Approximately $5k is dedicated to maintaining the RAIC non-profit entity that directs the AIID programming sponsored by a fiscal sponsor. The remaining funds are all dedicated to programmatic outputs, including staff time for programming and research.",
+                "link": "https://www.wakingup.com/foundation",
+                "dataCy": "wu-modal-click"
+            }
+        ]
+    },
+    {
+        "name": "Database Founding Sponsor",
+        "items": [
+            {
+                "name": "Partnership on AI",
+                "modalName": "PAI",
+                "text": "As a [[founding sponsor]] of the AI Incident Database (AIID), Partnership on AI supported the start-up of the database, including a grant to support the project prior to the founding of the Responsible AI Collaborative. Partnership on AI also invested in promoting the social benefits of the AIID to their Partners and other stakeholders and have an ongoing interest in supporting its outcomes.",
```

Comment:
It looks like you invented some syntax to be able to have links. Maybe use markdown and use the already existing `react-markdown` component?